lib/spider/spider/utils/sql.py

```diff
@@ -0,0 +1,38 @@
+import re
+import sqlite3
+
+from spider._path import DATASET_PATH
+
+
+def extract_sql(message: str, if_first_answer: bool = True) -> str:
```

Comment:
Can you add a docstring on this function to explain what it does, to improve readability & re-usability? (I guess this can be done with @svensundell)

Reply:
yes